test/expected.json

```diff
@@ -2692,16 +2692,17 @@ Resources:
               },
               "build": {
                 "commands": [
-                  "npm i && npm run bump",
-                  "aws secretsmanager get-secret-value --secret-id \"arn:aws:secretsmanager:us-east-1:712950704752:secret:delivlib/github-ssh-lwzfjW\" --output=text --query=SecretString > ~/.ssh/id_rsa",
-                  "chmod 0600 ~/.ssh/id_rsa",
-                  "BRANCH=bump/$(git describe)",
-                  "git branch -D $BRANCH || true",
-                  "git checkout -b $BRANCH",
-                  "git checkout master",
-                  "git merge $BRANCH",
-                  "git remote add origin_ssh git@github.com:awslabs/aws-delivlib-sample.git",
-                  "git push --follow-tags origin_ssh master"
+                  "git describe --exact-match HEAD && { echo \"No new commits.\"; export SKIP=true; } || { echo \"Changes to release.\"; export SKIP=false; }",
```

Comment:
rebase issue?

Reply:
It looks to me that the expectation is incorrect on the current `master`.